services/workflows-service/src/user/user.controller.internal.ts

```diff
@@ -38,7 +39,10 @@ export class UserControllerInternal {
   @swagger.ApiCreatedResponse({ type: [UserModel] })
   @UseGuards(AdminAuthGuard)
   @swagger.ApiForbiddenResponse()
-  async create(@common.Body() userCreatInfo: UserCreateDto) {
+  async create(
+    @common.Body() userCreatInfo: UserCreateDto,
+    @CurrentProject() currentProjectId: TProjectId,
```

Comment:
pick one, 
@CurrentProject() currentProjectId OR @CurrentProject() projectId.
I think contextProjectId or current is preferred